bin/VMSlogger

```diff
@@ -53,62 +55,119 @@ parser.add_argument(
     action="store_true",
     help="save into HDF 5. Requires h5py (pip install h5py)",
 )
-parser.add_argument("--chunk-size", dest="chunk_size", default=300000, type=int)
+parser.add_argument("--chunk-size", dest="chunk_size", default=5000, type=int)
```

Comment:
The `desp` here does not explain anything actually (the argument is `--chunk-size`).

Reply:
My fault, added. I also added epilog, providing more hints what this does.